modules/examples/health-check-fs2/health-client/src/main/scala/higherkindness/mu/rpc/healthcheck/client/HealthCheckClientHandler.scala

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2017-2019 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package higherkindness.mu.rpc.healthcheck.client
+
+import cats.effect.{Async, Resource, Sync}
+import higherkindness.mu.rpc.healthcheck.service.HealthCheckServiceFS2
+import cats.implicits._
+import cats.syntax._
+import higherkindness.mu.rpc.healthcheck._
+import higherkindness.mu.rpc.protocol.Empty
+import org.log4s.Logger
+
+class HealthCheckClientHandler[F[_]: Async](client: Resource[F, HealthCheckServiceFS2[F]])(
+    implicit logger: Logger) {
+
+  def updatingWatching(name: String) =
+    for {
+      _      <- client.use(_.setStatus(HealthStatus(HealthCheck(name), ServerStatus("SERVING"))))
+      status <- client.use(_.check(HealthCheck(name)))
+      _      <- Async[F].delay(logger.info("Status of " + name + " service update to" + status))
```

Comment:
I'd suggest using https://github.com/ChristopherDavenport/log4cats to prevent all the `Sync[F].delay(logger....)` here. What do you think?

Reply:
I think it's better. I use `log4s` because other modules like `client-cache` or `examples` use it. Maybe we should make a full change.

Reply:
I'll change from log4s to log4cats